### PR TITLE
Replacing jq with regex

### PR DESCRIPTION
**Pull request by RaynardGerraldo**
Replace jq with more conventional parsing #413

### CODE REVIEW

**Review by ura43 — CHANGES_REQUESTED** [review]
Please also modify the Readme to reflect the change

**Comment by ura43** [comment]
Can confirm functional with at least 3 anime

**Comment by RaynardGerraldo** [comment]
> Please also modify the Readme to reflect the change

will do

**Comment by RaynardGerraldo** [comment]
> Please also modify the Readme to reflect the change

done

**Review by ura43 — APPROVED** [review]

**Comment by 71zenith** [comment]
Why use multiple greps? 	`curl -s -H 'x-requested-with:XMLHttpRequest' "$ajax_url" -d "id=$ajax" -d "time=69420691337800813569" | sed -e 's/\].*/\]/' -e 's/\\\//g' | grep -Eo 'https?:\\/\\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)'
`

File: ani-cli [hunk 1]
```diff
@@ -167,7 +167,7 @@ decrypt_link() {
 	ajax=$(echo "$video_id" | openssl enc -aes256  -K "$secret_key" -iv "$iv" -a)
 
 	#send the request to the ajax url
-	curl -s -H 'x-requested-with:XMLHttpRequest' "$ajax_url" -d "id=$ajax" -d "time=69420691337800813569" | jq -r '.source[].file'
+	curl -s -H 'x-requested-with:XMLHttpRequest' "$ajax_url" -d "id=$ajax" -d "time=69420691337800813569"  | grep -Eo 'https?:\\\/\\\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)' | grep -oE "https:\\\/\\\/vidstreamingcdn.com.*&expiry=[0-9]+" | sed 's/\\//g' | sed '$d'
```
Comment (ura43) [inline ani-cli:170]: Make this at least 5 lines

**Review by ura43 — CHANGES_REQUESTED** [review]

**Comment by ura43** [comment]
> Why use multiple greps? `curl -s -H 'x-requested-with:XMLHttpRequest' "$ajax_url" -d "id=$ajax" -d "time=69420691337800813569" | sed -e 's/\].*/\]/' -e 's/\\\//g' | grep -Eo 'https?:\\/\\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)' `

Because readability and line length

**Comment by 71zenith** [comment]
> Because readability and line length

It probably slows down. About the line length, I can agree

**Comment by RaynardGerraldo** [comment]
> | sed -e 's/\].*/\]/' -e 's/\\\//g' | grep -Eo 'https?:\\/\\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)'

video url not found for me

**Comment by 71zenith** [comment]
you missed an asterisk in the first sed command

**Comment by RaynardGerraldo** [comment]
> you missed an asterisk in the first sed command

where exactly? i copied it from ur reply

**Comment by 71zenith** [comment]
sorry, i guess i missed something

**Comment by 71zenith** [comment]
This should work
sed -e 's/\].*/\]/' -e 's/\\//g' | grep -Eo 'https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)

**Comment by 71zenith** [comment]
Github deletes some stuff. Come on discord and I'll share there

File: ani-cli (hunk 1 above)
Comment (CoolnsX) [reply ani-cli:170]: replace vidstreaming with cdn.. because sometimes gogo servers gives the loadfast link instead of vidstreaming...

**Comment by RaynardGerraldo** [comment]
@ura43

**Review by ura43 — APPROVED** [review]

**Pull request merged**